### PR TITLE
x11-misc/qt5ct: new package

### PR DESCRIPTION
**Pull request by Coacher**
This program allows users to configure Qt5 settings (theme, font,
icons, etc.) under DE/WM without Qt5 integration.

Gentoo-Bug: 577840

Package-Manager: portage-2.3.0

### CODE REVIEW

File: x11-misc/qt5ct/qt5ct-0.24.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		ewarn "qt5ct configuration won't be applied to the currently running sessions."
+		ewarn "Please relogin."
+	fi
+	if ! has_version 'dev-qt/qtsvg:5'; then
+		echo
+		elog "For SVG icon themes, please install 'dev-qt/qtsvg:5'."
+		echo
+	fi
+
+	fdo-mime_desktop_database_update
```
Comment (kensington) [inline x11-misc/qt5ct/qt5ct-0.24.ebuild]: The installed .desktop file doesn't register any ```MimeType```.

File: x11-misc/qt5ct/qt5ct-0.24.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+HOMEPAGE="https://sourceforge.net/projects/qt5ct/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	dev-qt/qtcore:5=
```
Comment (kensington) [inline x11-misc/qt5ct/qt5ct-0.24.ebuild]: Does this package definitely use private parts of Qt? If it doesn't, the subslot operator is not required.

Comment (Coacher) [inline x11-misc/qt5ct/qt5ct-0.24.ebuild]: >Does this package definitely use private parts of Qt?

Yes.

File: x11-misc/qt5ct/qt5ct-0.24.ebuild (hunk 1 above)
Comment (Coacher) [inline x11-misc/qt5ct/qt5ct-0.24.ebuild]: Yes. Let me fix it.

**Comment by Coacher** [comment]
CC @gentoo/proxy-maint

**Comment by gktrk** [comment]
@gentoo/proxy-maint

**Comment by monsieurp** [comment]
Is it ready for merging?

**Comment by Coacher** [comment]
@monsieurp, from my point of view, yes it is.
@kensington, do you have any other objections or suggestions?

**Comment by Coacher** [comment]
Thank you, @monsieurp.

**Pull request closed without merging**